pkg/actors/actors.go

```diff
@@ -77,8 +77,8 @@ var (
 //nolint:interfacebloat
 type Actors interface {
 	Call(ctx context.Context, req *invokev1.InvokeMethodRequest) (*invokev1.InvokeMethodResponse, error)
-	Init() error
-	Stop()
+	Init(context.Context) error
+	Stop() error
```

Comment:
Sae, please keep errors here

```suggestion
	Stop() error
```